example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```

Comment:
should not be under source control :) please remove it

Reply:
removed.